js/documentation.js

```diff
@@ -0,0 +1,50 @@
+//Collapse presentations
+$("#presentation").click(function () {
+    $("#video").collapse('hide');
+    $("#pdf").collapse('show');
+
+});
+//Collapse videos
+$("#videos").click(function () {
+    $("#pdf").collapse('hide');
+    $("#video").collapse('show');
+});
+
+//Collapse pdf
+$('#pdf-list li').click(function () {
+    let id = $(this).find('a').attr('id');
+
+    if ($('#noCollapse').find(`#${id}`).length !== 1) {
+        let IDs = []
```

Comment:
```suggestion
        let IDs = [];
```

Reply:
Set semicolons to all the lines. I know it is not necessary in JS but it is a good practice and helps code review.